schemas/log.yml

```diff
@@ -156,3 +156,50 @@
 
         According to RFCs 5424 and 3164, the priority is 8 * facility + severity.
         This number is therefore expected to contain a value between 0 and 191.
+
+    - name: syslog.version
+      level: extended
+      type: long
+      example: 1
+      short: Syslog protocol version.
+      description: >
+        The version of the Syslog protocol specification. Only applicable for RFC 5424 messages.
+
+    - name: syslog.hostname
+      level: extended
+      type: keyword
+      example: example-host
+      short: The host that originated the Syslog message.
+      description: >
+        The hostname, FQDN, or IP of the machine that originally sent the Syslog message.
+
+    - name: syslog.appname
+      level: extended
+      type: keyword
+      example: sshd
+      short: The device or application that originated the Syslog message.
+      description: >
+        The device or application that originated the Syslog message, if available.
+
+    - name: syslog.procid
+      level: extended
+      type: keyword
+      example: 12345
+      short: The process name or ID that originated the Syslog message.
+      description: >
+        The process name or ID that originated the Syslog message, if available.
+
+    - name: syslog.msgid
+      level: extended
+      type: keyword
+      example: ID47
+      short: An identifier for the type of Syslog message.
+      description: >
+        An identifier for the type of Syslog message, if available. Only applicable for RFC 5424 messages.
+
+    - name: syslog.data
+      level: extended
+      type: flattened
+      short: Structured data expressed in RFC 5424 messages.
+      description: >
+        Structured data expressed in RFC 5424 messages, if available.
```

Comment:
How does the expected `flattened` object here look? Key-value pairs from the structure data?

I think it makes sense to add more guidance about which values will build out the `flattened` object captured here.

Reply:
Yes, it will be key-value pairs from the structured data portion. I made a mention to the RFC section as well.